### ISSUE TITLE
You should not oblige the user to be logged in to create an entry.

### ISSUE THREAD

**Issue by theodorDiaconu** [meteor-utilities/smart-methods#2]
Reasoning is simple:
If I have something that an anonymous user can insert, but not alter or delete afterwards, I cannot use your interesting plugin.

**Comment by SachaG** [comment]
That's true, but the idea behind this package is to make standard operations easier. I feel like anonymous inserting doesn't really count as a "standard operation" since it's not a very common use case…
